src/main/java/dev/jbang/cli/Jbang.java

```diff
@@ -62,13 +68,10 @@ public static CommandLine getCommandLine() {
 		@Override
 		public int handleExecutionException(Exception ex, CommandLine commandLine, CommandLine.ParseResult parseResult)
 				throws Exception {
+			Util.errorMsg(null, ex);
 			if (Util.isVerbose()) {
-				ex.printStackTrace();
```

Comment:
why remove this ?

Reply:
Again because `errorMsg()` already does the same thing. Thought it best to remove code duplication